node/hubController.js

```diff
@@ -3,15 +3,17 @@
 'use strict';
 
 var q = require('q');
-var hubsConfig = require("./hubsConfig");
+var HubsConfig = require("./hubsConfig");
 var OpenT2TError = require('opent2t').OpenT2TError;
 var OpenT2TConstants = require('opent2t').OpenT2TConstants;
+var OpenT2TLogger = require('opent2t').Logger;
 
 class HubController {
 
-    constructor() {
+    constructor(logLevel = "info") {
```

Comment:
So we can control verbosity - say if you run in a diagnostic environment you might want to pass in "verbose", in prod you might "info" OR you might want to tune it down to "warn" or "error"

---
In reply to: [103827843](https://github.com/openT2T/cloud/pull/26#discussion_r103827843) [](ancestors = 103827843)